internal/controllers/flavor/controller.go

```diff
@@ -27,19 +27,14 @@ import (
 	orcv1alpha1 "github.com/k-orc/openstack-resource-controller/api/v1alpha1"
 
 	ctrlexport "github.com/k-orc/openstack-resource-controller/internal/controllers/export"
+	"github.com/k-orc/openstack-resource-controller/internal/controllers/generic"
 	"github.com/k-orc/openstack-resource-controller/internal/scope"
 )
 
 const (
-	Finalizer = "openstack.k-orc.cloud/flavor"
-
 	FieldOwner = "openstack.k-orc.cloud/flavorcontroller"
```

Comment:
Possibly get this from generic?

Reply:
Needs more pieces before we can do that. They're coming.